view/frontend/web/js/checkout/shipping_method/show-myparcel-shipping-method.js

```diff
@@ -120,49 +121,58 @@ define(
                 typeof quote.shippingAddress._latestValue.street !== 'undefined' &&
                 typeof quote.shippingAddress._latestValue.street[0] !== 'undefined'
             ) {
-                var street0 = quote.shippingAddress._latestValue.street[0];
-                if (typeof street0 === 'undefined') street0 = '';
-                var street1 = quote.shippingAddress._latestValue.street[1];
-                if (typeof street1 === 'undefined') street1 = '';
-                var street2 = quote.shippingAddress._latestValue.street[2];
-                if (typeof street2 === 'undefined') street2 = '';
-                var country = quote.shippingAddress._latestValue.countryId;
-                if (typeof country === 'undefined') country = '';
-                var postcode = quote.shippingAddress._latestValue.postcode;
-                if (typeof postcode === 'undefined') postcode = '';
-                var city = quote.shippingAddress._latestValue.postcode;
-                if (typeof city === 'undefined') city = '';
+                address = getLoggedInAddress();
```

Comment:
getLoggedInAddress() kan je beter verbinden met een andere function. Net zoals je doet met MyParcel.foo() in myparcel.js. Anders kan dit een conflict geven met andere plugins. Doe dit ook voor `getNoLoggedInAddress`. (wel netjes zo in een andere function) 👍

Reply:
Zoals we hebben besproken, hoeft dit niet omdat het binnen dezelfde scope blijft.